migrations.lua

```diff
@@ -14,11 +16,14 @@ return {
   [2] = function()
     schema.create_table("packages", {
       { "id", types.serial {unique = true, primary_key = true}},
-      { "name", types.text {unique = true}},
+      { "name", types.text },
       { "version", types.text },
       { "description", types.text },
+      { "game", types.text },
```

Comment:
One package can apply to all or several games, though?

Reply:
My thought here is if your package is game specific, you would put the game here, and if it's generic you would leave it blank, since it's not for a specific game.

Reply:
We can't control how people will use this... I definitely foresee some trying to use it for many ;)

Reply:
fair, but since I think we're just gonna do a simple text search against it as an option I dunno that it's functionally much different?